jps-plugin/src/org/intellij/erlang/jps/builder/ErlangFileDescriptor.java

```diff
@@ -25,22 +24,20 @@
 
 import java.util.List;
 
-@Tag("module")
 public class ErlangFileDescriptor {
   @Attribute("path")
-  public String myErlangModulePath;
+  public String myPath;
 
   @Tag("dependencies")
   @AbstractCollection(surroundWithTag = false, elementTag = "dependency")
-  public List<String> myDependencies;
+  public List<String> myDependencies = ContainerUtil.newArrayList();
 
-  @SuppressWarnings("unused") //reflection
-  public ErlangFileDescriptor() {
-    myDependencies = ContainerUtil.newArrayList();
+  public ErlangFileDescriptor(@NotNull String path, @NotNull List<String> dependencies) {
+    myPath = path;
+    myDependencies = dependencies;
   }
 
-  public ErlangFileDescriptor(@NotNull String erlangModulePath, @NotNull List<String> dependencies) {
-    myErlangModulePath = erlangModulePath;
-    myDependencies = dependencies;
+  @SuppressWarnings("unused") // reflection
+  public ErlangFileDescriptor() {
   }
```

Comment:
What happened? Why reorder constructors? Everywhere in your code you have simpler constructors first, followed by more complex ones.